src/style.css

```diff
@@ -1,3 +1,76 @@
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  --active-color: #00acdc;
```

Comment:
good practice to declare variables inside the [:root](https://www.w3schools.com/css/css3_variables.asp) selector
```css
:root {
--active-color: #00acdc;
}
```